components/org.wso2.carbon.identity.api.user.association/org.wso2.carbon.identity.api.user.association.v1/src/main/resources/association.yaml

```diff
@@ -72,17 +76,19 @@ paths:
   /me/associations:
     post:
       description: |
-        This API is used to associate a user to the authenticated user. For example if it is required associate  my user account to the user account of 'john', this endpoint can be used. userId and the password are required to associate the accounts.
-
-         <b>Permission required:</b>
-         * /permission/admin/login
+        This API is used to associate a user to the authenticated user. For example if it is required to associate  my
```

Comment:
For example, if
The userid and password

Reply:
Changed